rules/precompiled_apple_resource_bundle.bzl

```
@@ -198,20 +198,28 @@ def _precompiled_apple_resource_bundle_impl(ctx):
 
     # See https://github.com/bazel-ios/rules_ios/pull/747 for context
     xccurrentversions = [
-        (None, None, depset([f]))
+        f
         for resource_files in ctx.attr.resources
         for f in resource_files.files.to_list()
         if f.extension == "xccurrentversion"
     ]
 
     return [
         AppleResourceInfo(
-            datamodels = xccurrentversions,
+            datamodels = [
+                (None, None, depset([f]))
+                for f in xccurrentversions
```

And this is meant to not be like this (what I poorly asked in the past PR), correct?
```suggestion
                (None, None, depset(xccurrentversions))
```

---

Oh I think it will be fine if it's written like this, and I def didn't have to write one `depset` per file 🤦. Will run a quick test and pull this in, thx for catching!

---

https://github.com/bazel-ios/rules_ios/pull/748/commits/94621fb0eeab1c4b2f01f30789c0127ea4729ccd